www/apps/book/app/learn/fundamentals/module-links/index-module/page.mdx

```diff
@@ -0,0 +1,487 @@
+export const metadata = {
+  title: `${pageNumber} Index Module`,
```

Comment:
comment: Don't make this change yet, I will need to discuss naming with Seb 😄 
```suggestion
  title: `${pageNumber} Index`,
```

Reply:
If we choose to go with `Index` instead of `the Index Module`, this will of cause need to be reflected across the docs.